cli/hzn.go

```diff
@@ -232,7 +235,7 @@ Environment Variables:
 
 	devServiceCmd := devCmd.Command("service", msgPrinter.Sprintf("For working with a service project."))
 	devServiceLogCmd := devServiceCmd.Command("log", msgPrinter.Sprintf("Show the container/system logs for a service."))
-	devServiceLogCmdServiceName := devServiceLogCmd.Flag("service", msgPrinter.Sprintf("The name of the service whose log records should be displayed. The service name is the same as the url field of a service definition.")).Short('s').String()
```

Comment:
Should keep this flag unchange, do not remove it. Issue https://github.com/open-horizon/anax/pull/2387 will change the flag to `-c`.

Reply:
Addressed this in recent changes